lib/kiba-common/transforms/source_transform_adapter.rb

```diff
@@ -3,7 +3,14 @@ module Common
     module Transforms
       class SourceTransformAdapter
         def process(args)
-          args.shift.new(*args).each do |row|
+          klass = args.shift
+          if args.last.is_a?(Hash) && RUBY_VERSION >= "2.7"
+            kwargs = args.pop
+            instance = klass.new(*args, **kwargs)
+          else
+            instance = klass.new(*args)
+          end
```

Comment:
For this case and all delegation cases, the only reliable way for current Ruby 2.7.0 & 2.7.1, is to use `ruby2_keywords def process(*args)`. That flags the keyword arguments, so once `*args` is used in the body they'll be keyword arguments again.

If you don't want to change the signature of `process(args)`, you'll need to find a caller which passes them as `*args` and use `ruby2_keywords` there. You can check if the keyword Hash is flagged correctly with `Hash.ruby2_keywords_hash?`.